package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "dagre": "^0.8.5"
```

Comment:
For some reason, this `package.json` (and associated `package-lock.json`) seems to have wound up in the root directory, instead of the `ui/` directory.

Also, we should stick to using yarn -- so to fix, you'll need to delete these files, go into the `ui` dir, and run `yarn add dagre`, and afterwards `yarn install` to be safe. Then commit `yarn.lock` and things should start working, I think!